charts/graphistry-helm/grafana-values.yaml

```diff
@@ -0,0 +1,48 @@
+grafana:
```

Comment:
maybe we carve out these into a plugins folder or something as well?

Reply:
put it in ~/overrides/plugins